readme.md

```diff
@@ -450,3 +461,36 @@ body, .foo {
   [RULE_END]  
 ]
 ```
+
+#### JavaScript Function Value
```

Comment:
Without being able to return ISTF or a simple string property/value we are not getting same abilities for postprocessors etc, essentially there is still at the end the need to parse CSS and this is exactly what we are tryting to avoid with ISTF.